### PR TITLE
Change network names

### PR DESCRIPTION
**Pull request by caramis**

### CODE REVIEW

**Review by psy2848048 — APPROVED** [review]

**Comment by jhlee-young** [comment]
please update a network name in `develop/dev-reference/history-and-changes` as well

File: content/docs/learn/glossary.md [hunk 1]
```diff
@@ -167,13 +167,13 @@ For more information on the Xpla core, see [Xpla Core Modules]({{< ref "core-mod
```
Comment (jhlee-young) [inline content/docs/learn/glossary.md]: Dimension-1 in line 108

File: content/docs/full-node/manage-a-validator/validator-faq.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ The active validator set is made up of 130 validators, who hold the most XPLA. T
```
Comment (jhlee-young) [inline content/docs/full-node/manage-a-validator/validator-faq.md:32]: still dimension-1 in line 29

**Comment by caramis** [comment]
Fixed !

**Review by jhlee-young — APPROVED** [review]

**Comment by caramis** [comment]
Rebase 가 안되어 있어서 rebase 후 머지하겠습니다.

**Pull request merged**